tests/integration-tests/tests/networking/test_placement_group/test_dynamic_placement_group_in_cluster/pcluster.config.ini

```diff
@@ -0,0 +1,28 @@
+[global]
+cluster_template = default
+
+[aws]
+aws_region_name = {{ region }}
+
+[cluster default]
+base_os = {{ os }}
+key_name = {{ key_name }}
+vpc_settings = parallelcluster-vpc
+scheduler = {{ scheduler }}
+master_instance_type = {{ instance }}
+compute_instance_type = {{ instance }}
+{% if scheduler == "awsbatch" %}
```

Comment:
I think the test will not run with batch? Let's remove the batch specific logic here

Reply:
Done